package.json

```diff
@@ -934,7 +934,8 @@
         "vue": "^2.5.16",
         "winston": "^3.2.1",
         "winston-transport": "^4.3.0",
-        "xml2js": "^0.4.19"
+        "xml2js": "^0.4.19",
+        "yaml": "^1.9.2"
```

Comment:
There is already a package reference to `js-yaml`, is it necessary to add another yaml package reference?

Reply:
From what I see, the existing package will allow user to use merges, aliases and other things (not sure exactly what) specified in 1.2 without throwing any errors. The user will think it is fine until they actually try to deploy it. At least according to the documentation, cloud formation / sam will error on encountering it.

Reply:
Thank you for pointing this out. I've made a work item to change the existing parsing code away from that 1.2 spec later on.

Reply:
It seems reasonable to add the yaml package, and we will remove the other package when we can